File: C/BEGINNER/2028 Sequence of Sequence.c
#include <stdio.h>

int main()
{
    int sequenceNum, count, caseNum, i, j;
    caseNum = 1;
    while (scanf("%d", &sequenceNum) != EOF)
    {
        if (sequenceNum == 0)
        {
            printf("Caso %d: 1 numero\n", caseNum);
            printf("0\n");
        }
        else if (sequenceNum == 1)
        {
            printf("Caso %d: 2 numeros\n", caseNum);
            printf("0 1\n");
        }
        else
        {
            count = 1;
            for (i = 1; i <= sequenceNum; i++)
            {
                for (j = i; j != 0; j--)
                {
                    count += 1;
                }
            }
            printf("Caso %d: %d numeros\n", caseNum, count);
            printf("0 ");
            for (i = 1; i <= sequenceNum; i++)
            {
                for (j = i; j != 0; j--)
                {
                    if ((j == 1) && (i == sequenceNum))
                        break;
                    printf("%d ", i);
                }
            }
            printf("%d\n", sequenceNum);
        }
        printf("\n");
        caseNum += 1;
    }
    return 0;
}

File: Python/BEGINNER/2028 Sequence of Sequence.py
case = 1
while True:
    try:
        sequence = []
        countSequence = 0
        sequenceSize = int(input())

        while True:
            if countSequence >= sequenceSize and sequenceSize != 0:
                break
            while countSequence < sequenceSize:
                sequence.append(sequenceSize)
                countSequence += 1
            sequenceSize -= 1
            countSequence = 0

        sequence.append(0)
        sortedSequence = sorted(sequence)
        sequenceString = (str(number) for number in sortedSequence)
        sequenceString = ' '.join(sequenceString)

    except EOFError:
        break
    else:
        if len(sortedSequence) <= 2:
            print('Caso %d: %d numero' % (case, len(sortedSequence)))
        else:
            print('Caso %d: %d numeros' % (case, len(sortedSequence)))
        print(sequenceString)
        case += 1
        print()